homework01/rudolf/vector.h

```diff
@@ -0,0 +1,31 @@
+#ifndef __VECTOR_H_INCLUDED__   // if x.h hasn't been included yet...
+#define __VECTOR_H_INCLUDED__   //   #define this so the compiler knows it has been included
+
+#include <iostream>
+
+class Vector
+{
+public:
+	Vector();
+
+	Vector(double x, double y);
+
+	double dotProduct(const Vector& vec) const;
+
+	double GetX() const;
+	double GetY() const;
+
+      	void Add(const Vector& v);
+
+      	// Overload = operator.
+      	void operator=(const Vector& v);
+	// Overload == operator.
+      	bool operator==(const Vector& v);
```

Comment:
It's time to spend some time in the indentation settings of your favorite IDE :)